tests/integration/conftest.py

```diff
@@ -420,6 +421,8 @@ def _disable_submission(schd: 'Scheduler') -> 'Set[TaskProxy]':
 
         def _submit_task_jobs(_, itasks, *args, **kwargs):
             nonlocal submitted_tasks
+            for itask in itasks:
+                itask.state_reset(TASK_STATUS_SUBMITTED)
```

Comment:
FYI: This fixture is intended to capture submission attempts, it wasn't intended to mock the submission process/outcome.

Reply:
I forget why I did that, checking back...